source/plugins/community/crypto/metadata.yml

```diff
@@ -0,0 +1,52 @@
+name: 🧩 Crypto
```

Comment:
Can you choose a different emoji as icon ? The puzzle one is used as a placeholder

Reply:
yes, i can